stack/schema.go

```diff
@@ -58,6 +58,23 @@ type Function struct {
 	Annotations *map[string]string `yaml:"annotations"`
 }
 
+type Configuration struct {
+	StackConfig StackConfiguration `yaml:"configuration"`
+}
+
+type StackConfiguration struct {
+	TemlpatesConfig TemplatesConfiguration `yaml:"template"`
+}
+
+type TemplatesConfiguration struct {
+	TemplateInfos []TemplateSource `yaml:"sources"`
```

Comment:
There was some brief discussion somewhere (i couldn't find it in #617) but I never thought we really resolved whether this should be a list versus a `map[string]string` or `map[string]TemplateSource`.  @alexellis said that we were guided to use a list somewhere else, but if we are going to immediately transform the list to a map anyway, why not just use a map to begin with?

Reply:
This is a very good proposal, will do it :+1:

Reply:
This is now just `map[string]string` as below the `templates` field :heavy_check_mark: